### PR TITLE
fix: add missing borderInShellHeader

### LINKED ISSUE #13128: [CAL-2887] make sure all settings <Meta> has borderInShellHeader props
and p-4 and border subtle for the content:

## like this

before (old layout):

CleanShot 2024-01-09 at 17 17 14@2x

after (new layout):

CleanShot 2024-01-09 at 17 18 01@2x

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2887](https://linear.app/calcom/issue/CAL-2887/make-sure-all-settings-meta-has-borderinshellheader-props)</sub>

### PR DESCRIPTION
**Pull request by anikdhabal**
/claim #13128
fixes #13128 

Apart from the calendars and appearance, all of the settings <meta> has borderinshellheader props.

eg:-

Screenshot 2024-01-10 002854

### CODE REVIEW

**Comment by anikdhabal** [comment]
cc @PeerRich

File: apps/web/pages/settings/my-account/appearance.tsx [hunk 1]
```diff
@@ -136,7 +136,7 @@ const AppearanceView = ({
 
   return (
     <div>
-      <Meta title={t("appearance")} description={t("appearance_description")} borderInShellHeader={false} />
+      <Meta title={t("appearance")} description={t("appearance_description")} borderInShellHeader={true} />
```
Comment (PeerRich) [inline apps/web/pages/settings/my-account/appearance.tsx:139]:
```suggestion
      <Meta title={t("appearance")} description={t("appearance_description")} borderInShellHeader />
```

File: apps/web/pages/settings/my-account/calendars.tsx [hunk 2]
```diff
@@ -44,11 +44,9 @@ const CalendarsView = () => {
         title={t("calendars")}
         description={t("calendars_description")}
         CTA={<AddCalendarButton />}
-        borderInShellHeader={false}
+        borderInShellHeader={true}
```
Comment (PeerRich) [inline apps/web/pages/settings/my-account/calendars.tsx:47]:
```suggestion
        borderInShellHeader
```

**Comment by PeerRich** [comment]
> Apart from the calendars and appearance, all of the settings has borderinshellheader props.

i think there are team settings and org settings that still miss them

**Comment by anikdhabal** [comment]
> > Apart from the calendars and appearance, all of the settings has borderinshellheader props.
> 
> i think there are team settings and org settings that still miss them

ohh,I forget it. let me fix that quickly.

**Comment by anikdhabal** [comment]
@PeerRich Pls give a review now.

**Review by Udit-takkar — COMMENTED** [review]
@PeerRich on figma https://www.figma.com/file/UajFu4M1APhn2AywAEJkJr/New-Features?node-id=3373%3A174232&mode=dev not every setting page has any border.

Screenshot 2024-01-10 at 1 56 06 PM

**Comment by anikdhabal** [comment]
> @PeerRich on figma https://www.figma.com/file/UajFu4M1APhn2AywAEJkJr/New-Features?node-id=3373%3A174232&mode=dev not every setting page has any border.
> 
> Screenshot 2024-01-10 at 1 56 06 PM

Do I have to follow the Figma design, or can I keep it as it is? @PeerRich @Udit-takkar 

Don't know about the calendar settings, but I think the appearance is fine without a border.

**Comment by PeerRich** [comment]
ohh i see. in that case follow the figma

**Comment by anikdhabal** [comment]
Hey @PeerRich, @Udit-takkar, if we follow the Figma design, then everything currently adheres to the design. So, is this issue relevant?

Or, I think adding borders to some settings would make it perfect. eg:-

Screenshot 2024-01-18 121207
Screenshot 2024-01-18 120319

**Comment by PeerRich** [comment]
@anikdhabal yeah i like those changes you made there. is the PR ready for this?

**Comment by anikdhabal** [comment]
> @anikdhabal yeah i like those changes you made there. is the PR ready for this?

Yeah, let me quickly make those changes and share the result.

**Comment by anikdhabal** [comment]
> @anikdhabal yeah i like those changes you made there. is the PR ready for this?

This Pr is now ready for those.

**Comment by anikdhabal** [comment]
cc @Udit-takkar

**Review by supalarry — COMMENTED** [review]
Thank you for your contribution. Here are some things I noticed:

First, Admin settings. OAuth does have the border, but Features, Impersonation, Users, Organizations do not.

Screenshot 2024-02-07 at 10 07 20

---

Next, I think that after having added border to Apps this section should be centered:
Screenshot 2024-02-07 at 10 03 19

---

Third, Calendars and Appearance have space at the top in the designs but not in code

Screenshot 2024-02-07 at 10 05 10

Screenshot 2024-02-07 at 10 15 03

---

I think it's best to ask Ciaran about this - will send him this PR. Personally, I would have borders for all Admin categories, add margin top for Appearance and Calendars and center the Apps section.

@ciaranha can you please let us know which pages should have the border and which not?

**Comment by itsberkaya** [comment]
Actually we don't have borders only for Calendars and Appearance. We have borders for everything else. The only thing that needs to change in the app is the top space.

**Comment by anikdhabal** [comment]
Currently, I have no clue where to add or where not to.

**Comment by supalarry** [comment]
> Currently, I have no clue where to add or where not to.

@anikdhabal, @itsberkaya is designer at Cal, so we can refer to what he wrote:

> Actually we don't have borders only for Calendars and Appearance. We have borders for everything else. The only thing that needs to change in the app is the top space.

So the requirements are:
1. Have border for everything except Calendars and Appearance.
2. Just like in the Figma designs above, make sure there is space at the top for Calendars and Apperance.

Thank you for your contributions and let us know if you have any questions : )

**Comment by anikdhabal** [comment]
> > Currently, I have no clue where to add or where not to.
> 
> @anikdhabal, @itsberkaya is designer at Cal, so we can refer to what he wrote:
> 
> > Actually we don't have borders only for Calendars and Appearance. We have borders for everything else. The only thing that needs to change in the app is the top space.
> 
> So the requirements are:
> 
> 1. Have border for everything except Calendars and Appearance.
> 2. Just like in the Figma designs above, make sure there is space at the top for Calendars and Apperance.
> 
> Thank you for your contributions and let us know if you have any questions : )

Thank for your review. Let me fix that quickly.

**Comment by supalarry** [comment]
@anikdhabal thanks for the fast response! The thing is that currently, this PR consists only of added margin and no updates to the border:

Screenshot 2024-02-13 at 09 42 32

**Comment by PeerRich** [comment]
this has been addressed elsewhere

**Pull request closed without merging**